tests/chunked-encode-large.rs

```diff
@@ -84,17 +84,21 @@ async fn chunked_large() -> Result<(), http_types::Error> {
 
     let client = task::spawn(async move {
         task::sleep(Duration::from_millis(100)).await;
-        let mut res = surf::get(format!("http://localhost:{}", port)).await?;
+        let mut res = surf::get(format!("http://localhost:{}", port))
+            .await
+            .unwrap();
```

Comment:
Why do this over `?`?

Reply:
There's a type mismatch between http_types::result and error in this crate and in surf's crate